src/store/inputSlice.js

```diff
@@ -115,6 +120,18 @@ const inputSlice = createSlice({
       if (index !== -1)
         state.keyboard.pressedKeys.splice(index, 1);
       computeInput(prevState, state, action);
+    },
+
+    enableIK(state, action) {
+      const { enable } = action.payload;
+      state.inverseKinematics.enabled = enable;
+      if (enable) {
+        state.computed.arm.elbow = 0;
+        state.computed.arm.shoulder = 0;
+      } else {
+        state.computed.arm.ikForward = 0;
+        state.computed.arm.ikUp = 0;
+      }
```

Comment:
The input handler will take care of this, so we don't need to set these to 0 here.